tests/v1/kv_connector/unit/test_nixl_connector.py

```diff
@@ -378,10 +329,14 @@ def test_concurrent_load_kv(
         raise TimeoutError("Took too long to complete async handshake.")
 
 
+# NOTE: resource cleanup in mp backend is a bit finicky, so the order in which
```

Comment:
@kouroshHakha could you elaborate on the resource cleanup problem with mp. Are there processes left running?

Reply:
yeah the second parametrization regardless of whether it's "mp" or "ray" hits an OOM issue on gpu. I tried using `vllm.distributed.cleanup_dist_env_and_memory` but didn't quite work.

Reply:
I realized this test originally did not run on mp, distributed_executor_backend used to be None so I changed the latest version to None to only add ray to what was covered before.